crates/topos/src/components/regtest/commands/push_certificate.rs

```diff
@@ -3,6 +3,7 @@ use clap::Args;
 use crate::options::input_format::InputFormat;
 
 #[derive(Args, Debug)]
+#[command(about = "Push a certificate to a TCE node")]
```

Comment:
```suggestion
#[command(about = "Push a certificate to a TCE process")]
```

nit on the terminology, just to not let users think that they can run one "TCE node" somewhere, but instead can reach the "TCE process" of one Topos subnet node